src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs

```diff
@@ -144,13 +151,23 @@ await functionMetadata
                 await FileUtility.WriteAsync(dataFilePath, functionMetadata.TestData);
             }
 
+            var updatedFunctionMetadata = (await _hostFunctionMetadataProvider.GetFunctionMetadataAsync(_languageWorkerOptions.CurrentValue.WorkerConfigs, configChanged))
```

Comment:
nit: maybe just "functionMetadata"

Reply:
already have one with that name in the scope